build.gradle

```diff
@@ -2051,7 +2060,7 @@ project(":graphics") {
     project.ext.includeSources = true
     project.ext.moduleRuntime = true
     project.ext.moduleName = "javafx.graphics"
-    project.ext.mavenPublish = true
+    project.ext.mavenPublish = IS_MAVEN_PUBLISH
```

Comment:
Is this property needed? None of the other projects define it, and I don't see any usage of it.

Reply:
Yes, it doesn't seem to be used anywhere. I would indeed remove it.